src/reporter/index.ts

```diff
@@ -347,11 +347,15 @@ export default class Reporter {
     }
 
     private static _createTestRunInfo (reportItem: TestInfo): TestRunInfo {
+        const durationMs = !reportItem.test.skip
```

Comment:
I propose moving this code to the separate function and split it to the better redability

```js
_getTestDuration (reportItem: TestInfo): number {
    if (reportItem.test.skip)
        return 0;

    return +new Date() - (reportItem.startTime as number);
}
```